metrics_utils.py

```diff
@@ -25,11 +97,10 @@ def __call__(self, groundtruth, prediction, **kwargs):   # assuming 4D tensor is
 
 
 class OneHotFalseNegatives(FalseNegatives):
-    def __init__(self, thresholds=None, name=None, dtype=None):
-        self.name = name or 'FN_1H'
+    def __init__(self, thresholds=None, name='FN_1H', dtype=None):
         super().__init__(
             thresholds=thresholds,
-            name=self.name,
+            name=name,
```

Comment:
what does name=name do here?

Reply:
(happens many times throughout the file)

Reply:
same thing with dtype

Reply:
this one is my bad, I misread the code

Reply:
resolved, needed in tf2 due to class instance tracking

Reply:
maintained